src/chatdbg/pdb_util/locals.py

```diff
@@ -82,7 +82,7 @@ def _repr_if_defined(obj: Any) -> bool:
     return result
 
 
-def _format_limited(value: Any, limit: int = 10, depth: int = 3) -> str:
+def _format_limited(value: Union[int, np.ndarray], limit: int = 10, depth: int = 3) -> str:
```

Comment:
This one is a bit sketchy.  I explicitly put `Any` in because this function must work on any value, but typerwriter narrowed it to just the types seen in the test runs.

Reply:
That's what it does! :)